### PR TITLE
EAPPS-445 - Add a doc page for Edge Functions

### PR DESCRIPTION
**Pull request by phdunham**

### CODE REVIEW

**Comment by tristanlee85** [comment]
@phdunham, I talked this over with @jsernaque and we think that the Edge Functions should be in its own guide under CDN-as-Code, and keep any UI-related content in the file you modified.

I'll make these changes and will merge when ready. Thanks for the PR!

File: src/config/v7.nav.js [hunk 1]
```diff
@@ -85,10 +85,10 @@ const nav = {
               title: 'Route Features',
               path: 'performance/cdn_as_code/route_features',
             },
-            // {
-            //   title: 'Edge Functions',
-            //   path: 'performance/cdn_as_code/edge_functions',
-            // },
+            {
+              title: 'Edge Functions',
+              path: 'edge-functions',
```
Comment (jsernaque) [inline src/config/v7.nav.js:90]:
```suggestion
              path: 'performance/cdn_as_code/edge-functions',
```

File: src/guides/v7/edge-functions.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+| `context`                       | Object          | A read-only object providing additional information about the request and your environment                                                    |                                                                                                          |
+| `context.client`                | Key-value store | The client's network information                                                                                                              | `virt_` variables in [Feature Variables](/guides/performance/rules/feature_variables)                    |
+| `context.device`                | Key-value store | The client's device capabilities                                                                                                              | `wurlf_` variables in [Feature Variables](/guides/performance/rules/feature_variables)                   |
```
Comment (jsernaque) [inline src/guides/v7/edge-functions.md:39]:
```suggestion
| `context.device`                | Key-value store | The client's device capabilities                                                                                                              | `wurfl_` variables in [Feature Variables](/guides/performance/rules/feature_variables)                   |
```

File: src/guides/v7/edge-functions.md [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+| `context.environmentVars`       | Key-value store | Environment variables as defined in the {{ PRODUCT }} console under the Property -> Environment -> Environment Variables section              | [Environment Variables](/guides/basics/environments#environment-variables)                               |
```
Comment (jsernaque) [inline src/guides/v7/edge-functions.md:40]:
```suggestion
| `context.environmentVars`       | Key-value store | Environment variables as defined in the {{ PORTAL }} (*Property* -> **Environments** -> *Environment* --> **Environment Variables**)               | [Environment Variables](/guides/basics/environments#environment-variables)                               |
```

File: src/guides/v7/edge-functions.md [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+| `context.geo`                   | Key-value store | The client's geo location                                                                                                                     | `geo_` variables in [Feature Variables](/guides/performance/rules/feature_variables)                     |
+| `context.metrics`               | Object          | Provides functions for injecting metrics into your edge function                                                                              | [Access Logs](guides/logs/access_logs)                                                                   |
+| `context.origins`               | Key-value store | Origin servers as defined in the {{ PRODUCT }} console under the Property -> Environment -> Origins section (or the `{{ CONFIG_FILE }}` file) | [Origin Configuration](/guides/basics/hostnames_and_origins#origin)                                      |
```
Comment (jsernaque) [inline src/guides/v7/edge-functions.md:43]:
```suggestion
| `context.origins`               | Key-value store | Origin servers as defined in the {{ PORTAL }} (*Property* -> *Environment* -> **Origins**) or the `{{ CONFIG_FILE }}` file | [Origin Configuration](/guides/basics/hostnames_and_origins#origin)                                      |
```

File: src/guides/v7/edge-functions.md [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+| `context.environmentVars`       | Key-value store | Environment variables as defined in the {{ PRODUCT }} console under the Property -> Environment -> Environment Variables section              | [Environment Variables](/guides/basics/environments#environment-variables)                               |
+| `context.geo`                   | Key-value store | The client's geo location                                                                                                                     | `geo_` variables in [Feature Variables](/guides/performance/rules/feature_variables)                     |
+| `context.metrics`               | Object          | Provides functions for injecting metrics into your edge function                                                                              | [Access Logs](guides/logs/access_logs)                                                                   |
```
Comment (jsernaque) [inline src/guides/v7/edge-functions.md:42]:
```suggestion
| `context.metrics`               | Object          | Provides functions for injecting metrics into your edge function                                                                              | [Access Logs](/guides/logs/access_logs)                                                                   |
```

**Pull request merged**